requirements.txt

```diff
@@ -1,28 +1,532 @@
--e git+git://github.com/seung-lab/PyChunkedGraph.git@vminnie-90#egg=pychunkedgraph
-dynamicannotationdb>=2.0.1
-emannotationschemas>=3.1.4
+#
+# This file is autogenerated by pip-compile
```

Comment:
Is pip-compile still a requirement for this PR? Did we find what requirement caused the issue?

Reply:
oh i guess no its not a requirement, I do think this is safer than what we had before.

Reply:
i mean.. because there is a requirements.txt file in the repo you don't need pip-compile to build it or install it.   But we could use pip-compile to start managing upgrading our requirements in a more intentional way.